header.php

```diff
@@ -57,7 +58,7 @@
     if(count($data) > 0)
     {
         foreach ($data as $line) {
-            list($key, $val) = explode(":", $line);
+            @list($key, $val) = explode(":", $line);
```

Comment:
What errors would this generate? If we can we should check for them instead of ignoring.